pkg/common/cns-lib/volume/manager.go

```diff
@@ -951,3 +956,54 @@ func (m *defaultManager) ConfigureVolumeACLs(ctx context.Context, spec cnstypes.
 	}
 	return err
 }
+
+// RegisterDisk registers a virtual disk as a First Class Disk.
+// This method helps in registering VCP volumes as FCD using vslm endpoint
+func (m *defaultManager) RegisterDisk(ctx context.Context, path string, name string) (string, error) {
```

Comment:
shouldn't these go into govmomi repo?

Reply:
No, this is just a wrapper to return VStorageObject object Id. This also should not go to govmomi

Reply:
FYR, https://github.com/vmware/govmomi/pull/2319#discussion_r593943703